### PR TITLE
Maintenance/node workspace

### PR DESCRIPTION
**Pull request by tvanlaerhoven**
- Use node workspaces with shared node_modules 
- Use changsets to track changelog
- Update github workflows
- Add documentation pages (to be inserted in main theoplayer docs)

### CODE REVIEW

File: youbora/package.json [hunk 1]
```diff
@@ -52,29 +52,16 @@
...
-    "react-native-builder-bob": "^0.23.2",
-    "react-native-theoplayer": "^7.0.0",
-    "theoplayer": "^7.0.0",
-    "typescript": "^4.9.5",
-    "youboralib-util": "^1.4.3"
+  "dependencies": {
+    "youbora-adapter-theoplayer2": "^6.8.10",
+    "youboralib": "^6.8.54"
   },
-  "peerDependencies": {
```
Comment (MattiasBuelens) [inline youbora/package.json:73]: I think this is a mistake? You still want React and THEOplayer to be peer dependencies.

File: youbora/package.json [hunk 2]
```diff
@@ -30,10 +30,10 @@
   ],
   "scripts": {
     "typescript": "tsc --noEmit",
-    "lint": "eslint \"**/*.{ts,tsx}\"",
-    "prepare": "npm run manifest && bob build",
+    "build": "npm run manifest && bob build",
+    "prepare": "npm run build",
     "manifest": "youbora-util manifest",
```
Comment (MattiasBuelens) [inline youbora/package.json:35]: This script seems to be regenerating `youbora/manifest.json` on every `npm install`. This writes the current date to `"built"` ([source](https://github.com/THEOplayer/react-native-theoplayer-analytics/blob/b8e7acfb0c0dc69a95bd881ff23afce42627c464/youbora/manifest.json#L7)), which [pollutes the Git history a bit](https://github.com/THEOplayer/react-native-theoplayer-analytics/commits/main/youbora/manifest.json).

* Is `manifest.json` supposed to be an auto-generated file? If so, it should be in `.gitignore`.
* Is `manifest.json` supposed to be included in the [npm package](https://www.npmjs.com/package/@theoplayer/react-native-analytics-youbora)? If so, it should be listed in `package.json` under `"files"`.

File: youbora/package.json (hunk 1 above)
Comment (tvanlaerhoven) [reply youbora/package.json:73]: True, I'll fix it.

File: youbora/package.json (hunk 2 above)
Comment (tvanlaerhoven) [reply youbora/package.json:36]: 2x correct. It should be part of the packed files indeed.
The same for agama, which also has a generated manifest.

**Review by MattiasBuelens — APPROVED** [review]

**Review by wvanhaevre — APPROVED** [review]

**Pull request merged**